README.md

```diff
@@ -207,7 +207,7 @@ add_filter( 'wp_sentry_public_dsn', 'customize_public_sentry_dsn' );
 
 #### `wp_sentry_public_options` (array)
 
-You can use this filter to customize/override the sentry options used to initialize the JS tracker.
+You can use this filter to customize/override the [Sentry options](https://docs.sentry.io/error-reporting/configuration/) used to initialize the JS tracker.
```

Comment:
```suggestion
You can use this filter to customize/override the Sentry [options](https://docs.sentry.io/error-reporting/configuration/?platform=browser#common-options) used to initialize the JS tracker.
```

^ a bit more specific with the URL to prevent a different language (like Rust for me when I clicked) from showing